### PR TITLE
*: add configuration performance lite-init-stats 

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

as tittle

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/13719
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -548,6 +548,15 @@ Configuration items related to performance.
 + Default value: `1000`
 + Currently, the valid value range is `[1, 100000]`.
 
+### `lite-init-stats` <span class="version-mark">New in v7.1.0</span>
+
+> **Warning:**
+>
+> This variable is an experimental feature. It is not recommended to use it in production environments. If you find a bug, you can file an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:555]:
```suggestion
> This variable is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by songrijie — APPROVED** [review]
LGTM

**Review by xuyifangreeneyes — APPROVED** [review]

**Comment by xuyifangreeneyes** [comment]
PTAL @hfxsd @songrijie @Oreoxmt

File: statistics.md [hunk 2]
```diff
@@ -723,6 +723,12 @@ After enabling the synchronously loading statistics feature, you can further con
...
 
+> **Warning:**
+>
+> Lightweight statistics initialization is an experimental feature. It is not recommended that you use it in the production environment. This feature might be changed or removed without prior notice. If you find a bug, you can report an [issue](https://github.com/pingcap/tidb/issues) on GitHub.
+
+Since v7.1.0. TiDB introduces lightweight statistics initialization. When the value of `lite-init-stats` is true, statistics initialization doesn't load any histogram, TopN, Count-Min Sketch of any index or column into memory. When the value of `lite-init-stats` is false, statistics initialization loads histograms, TopN, Count-Min Sketch of indexes and primary keys into memory but doesn't load any histogram, TopN, Count-Min Sketch of any non-primary key column into memory. When the optimizer needs the histogram, TopN, Count-Min Sketch of some index or column, those needed statistics will be loaded into memory synchronously or asynchronously. Setting `lite-init-stats` to true speeds up statistics initialization and reduces TiDB's memory usage by avoiding unnecessary statistics loading.
```
Comment (hfxsd) [inline statistics.md:730]:
```suggestion
Starting from v7.1.0. TiDB introduces [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) for lightweight statistics initialization. 

- When the value of [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) is `true`, statistics initialization does not load any histogram, TopN, Count-Min Sketch of any index or column into memory. 
- When the value of `lite-init-stats` is `false`, statistics initialization loads histograms, TopN, Count-Min Sketch of indexes and primary keys into memory but does not load any histogram, TopN, Count-Min Sketch of any non-primary key column into memory. When the optimizer needs the histogram, TopN, Count-Min Sketch of some index or column, those needed statistics will be loaded into memory synchronously or asynchronously. 

Setting `lite-init-stats` to `true` speeds up statistics initialization and reduces TiDB memory usage by avoiding unnecessary statistics loading.
```

File: statistics.md [hunk 3]
```diff
@@ -723,6 +723,17 @@ After enabling the synchronously loading statistics feature, you can further con
...
+
+Starting from v7.1.0. TiDB introduces [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) for lightweight statistics initialization. 
+
+- When the value of [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) is `true`, statistics initialization does not load any histogram, TopN, Count-Min Sketch of any index or column into memory. 
+- When the value of `lite-init-stats` is `false`, statistics initialization loads histograms, TopN, Count-Min Sketch of indexes and primary keys into memory but does not load any histogram, TopN, Count-Min Sketch of any non-primary key column into memory. When the optimizer needs the histogram, TopN, Count-Min Sketch of some index or column, those needed statistics will be loaded into memory synchronously or asynchronously. 
+
+Setting `lite-init-stats` to `true` speeds up statistics initialization and reduces TiDB memory usage by avoiding unnecessary statistics loading.
```
Comment (hfxsd) [inline statistics.md:735]:
```suggestion
The default value is `false`, which means to disable lightweight statistics initialization. Setting `lite-init-stats` to `true` speeds up statistics initialization and reduces TiDB memory usage by avoiding unnecessary statistics loading.
```

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -548,6 +548,17 @@ Configuration items related to performance.
...
+
++ Controls whether to use lightweight statistics initialization during TiDB startup.
++ Default value: false
++ When the value of `lite-init-stats` is true, statistics initialization doesn't load any histogram, TopN, Count-Min Sketch of any index or column into memory. When the value of `lite-init-stats` is false, statistics initialization loads histograms, TopN, Count-Min Sketch of indexes and primary keys into memory but doesn't load any histogram, TopN, Count-Min Sketch of any non-primary key column into memory. When the optimizer needs the histogram, TopN, Count-Min Sketch of some index or column, those needed statistics will be loaded into memory synchronously(see [`tidb_stats_load_sync_wait`](/system-variables.md#tidb_stats_load_sync_wait-new-in-v540)) or asynchronously.
++ Setting `lite-init-stats` to true speeds up statistics initialization and reduces TiDB's memory usage by avoiding unnecessary statistics loading. For details, see [Load statistics](/statistics.md#load-statistics).
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:560]:
```suggestion
+ When the value of `lite-init-stats` is `true`, statistics initialization does not load any histogram, TopN, or Count-Min Sketch of indexes or columns into memory. When the value of `lite-init-stats` is `false`, statistics initialization loads histograms, TopN, and Count-Min Sketch of indexes and primary keys into memory but does not load any histogram, TopN, or Count-Min Sketch of non-primary key columns into memory. When the optimizer needs the histogram, TopN, and Count-Min Sketch of a specific index or column, the necessary statistics are loaded into memory synchronously or asynchronously (controlled by [`tidb_stats_load_sync_wait`](/system-variables.md#tidb_stats_load_sync_wait-new-in-v540)).
+ Setting `lite-init-stats` to `true` speeds up statistics initialization and reduces TiDB memory usage by avoiding unnecessary statistics loading. For details, see [Load statistics](/statistics.md#load-statistics).
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**